modules/ROOT/pages/push.adoc

```diff
@@ -0,0 +1,71 @@
+// start {partialsdir}/attributes.adoc[]
+include::_partials/attributes.adoc[]
+// end {partialsdir}/attributes.adoc[]
+
+:toclevels: 4
+
+:toc:
+
+// For more information, see: https://redhat-documentation.github.io/modular-docs/
+
+:context: {unifiedpush-service}
+ifdef::context[:parent-context: {context}]
+
+[id='a-collection-of-modules']
+= Using the {unifiedpush-service} Mobile Service
+
+The {unifiedpush-service} service allows you to send native push notifications to different mobile operating systems.
```

Comment:
This text is under toc without a heading - bad readability. A reason why the heading was removed here?